docs/en/guide/deploy.md

```diff
@@ -214,6 +214,7 @@ Don't enable options like _Auto Minify_ for HTML code. It will remove comments f
        # - apk add git # Uncomment this if you're using small docker images like alpine and have lastUpdated enabled
        - npm install
        - npm run docs:build
+       - mv docs/.vitepress/dist public # to allow gitlab pages to access the artifact
```

Comment:
see step 1

Reply:
My gitlab-ci then isnt working as expected.. please test then comment.

Reply:
Your directory structure might be something different. Can you show screenshots of your directory structure, your build commands and your vitepress config.